app/assets/stylesheets/modules/spotlight_overrides.scss

```diff
@@ -220,3 +225,8 @@ h5,
 #skip-link {
   position: relative !important;
 }
+
+// Allows for emphasized link weight for 'Spotlight at Stanford'
+.nav-link.emphasized {
+  font-weight: 500;
```

Comment:
Can we just use the pre-existing `fw-medium` Bootstrap class for this?

Reply:
Yes. Sounds good.